doc/source/user_guide/fields/field_data.rst

```diff
@@ -99,50 +92,46 @@ You can call the ``get_scalar_field_data`` method to get scalar field data, such
 
 .. code-block:: python
 
-  >>> abs_press_data = field_data.get_scalar_field_data(field_name="absolute-pressure", surface_name="cold-inlet")
-  >>> abs_press_data.size
-  241
-  >>> abs_press_data[120].scalar_data
+  >>> abs_press_data = field_data.get_scalar_field_data(field_name="absolute-pressure", surfaces=["cold-inlet"])
+  >>> abs_press_data["cold-inlet"].shape
+  (241,)
+  >>> abs_press_data["cold-inlet"][120]
   101325.0
 
-If a surface name is provided as input, scalar field data is returned.
-If surface IDs are provided as input, a dictionary containing a map of surface IDs to scalar field data is returned.
+If a single surface is provided as input, scalar field data is returned.
+If multiple surfaces are provided as input, a dictionary containing a map of surface IDs to scalar field data is returned.
 
 Get vector field data
 ~~~~~~~~~~~~~~~~~~~~~
 You can call the ``get_vector_field_data`` method to get vector field data.
 
 .. code-block:: python
 
-  >>> velocity_vector_data = field_data.get_vector_field_data(field_name="velocity", surface_name="cold-inlet")
-  >>> velocity_vector_data.size
-  152
-  >>> velocity_vector_data.scale
-  1.0
+  >>> velocity_vector_data = field_data.get_vector_field_data(field_name="velocity", surfaces=["cold-inlet"])
+  >>> velocity_vector_data["cold-inlet"].shape
+  (152, 3)
 
-If a surface name is provided as input, vector field data is returned.
-If surface IDs are provided as input, a dictionary containing a map of surface IDs to vector field data is returned.
+If a single surface is provided as input, vector field data is returned.
+If multiple surfaces are provided as input, a dictionary containing a map of surface IDs to vector field data is returned.
 
 Get pathlines field data
 ~~~~~~~~~~~~~~~~~~~~~~~~
 You can call the ``get_pathlines_field_data`` method to get pathlines field data.
 
 .. code-block:: python
 
-  >>> path_lines_data = field_data.get_pathlines_field_data(field_name="velocity", surface_name="cold-inlet")
-  >>> path_lines_data["vertices"].size
-  76152
-  >>> path_lines_data["lines"].size
+  >>> path_lines_data = field_data.get_pathlines_field_data(field_name="velocity", surfaces=["cold-inlet"])
+  >>> path_lines_data["cold-inlet"]["vertices"].shape
```

Comment:
@prmukherj There is an inconsistency between the pathlines data and surface data return structure:

pathlines_data[<surface>][<vertices>]
surface_data[<vertices>][<surface>]

`<surface>` should preferably be the first index of the returning array for all APIs.

Reply:
Done, Thank you.